app/templates/components/notification-dropdown.hbs

```diff
@@ -0,0 +1,59 @@
+    <div class="ui wide notification popup bottom lef transition ">
+  <div class="ui fluid link  divided list">
+    <div class="item">
+      <div class="right floated content">
+        <a>{{t 'Mark all as Read'}}</a>
+      </div>
+    </div>
+    <div class="item">
+      <div class="content">
+        <a class="header"><span>New Session Propoal</span></a>
```

Comment:
No individual anchor elements inside a notification. The whole notification should be clickable. (See facebook for example)